scripts/linters/js_ts_linter.py

```diff
@@ -1139,6 +1139,55 @@ def _check_comments(self):
             summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_broadcast_usage(self):
```

Comment:
Can we use general_purpose_linter.BAD_PATTERNS_JS_AND_TS_REGEXP instead of this check function?

[I would have recommended writing it as a custom plugin for eslint but the structure for writing eslint_puling is not ready yet!]